### PR TITLE
KAA-945 Check long-running-tests

### PR DESCRIPTION
**Pull request by Dubland**
/cc @rasendubi @forGGe

### CODE REVIEW

File: client/client-multi/client-c/test/platform-impl/test_ext_log_upload_strategies.c [hunk 1]
```diff
@@ -197,7 +199,7 @@ void test_upload_decision_by_timeout(void **state)
 
     log_storage_context.upload_timeout = DEFAULT_UPLOAD_TIMEOUT_THRESHOLD + 1;
 
-    sleep(1);
```
Comment (rasendubi) [inline client/client-multi/client-c/test/platform-impl/test_ext_log_upload_strategies.c:200]: That should be `DEFAULT_UPLOAD_TIMEOUT_THRESHOLD`

File: client/client-multi/client-c/test/platform-impl/test_ext_log_upload_strategies.c [hunk 2]
```diff
@@ -241,7 +243,6 @@ void test_upload_decision_on_failure(void **state)
 
     kaa_error_t error_code = KAA_ERR_NONE;
 
-    size_t DEFAULT_RETRY_PERIOD            = 2;
```
Comment (rasendubi) [inline client/client-multi/client-c/test/platform-impl/test_ext_log_upload_strategies.c:244]: Move this back.

File: client/client-multi/client-c/test/platform-impl/test_ext_log_upload_strategies.c [hunk 3]
```diff
@@ -33,6 +33,8 @@
 
 #include "platform-impl/common/ext_log_upload_strategies.h"
 
+#define DEFAULT_RETRY_PERIOD 1
```
Comment (rasendubi) [inline client/client-multi/client-c/test/platform-impl/test_ext_log_upload_strategies.c]: Prefer local const variables over defines

File: client/client-multi/client-c/test/platform-impl/test_ext_log_upload_strategies.c [hunk 4]
```diff
@@ -240,8 +240,8 @@ void test_upload_decision_on_failure(void **state)
     (void)state;
 
     kaa_error_t error_code = KAA_ERR_NONE;
-
-    size_t DEFAULT_RETRY_PERIOD            = 2;
+
```
Comment (rasendubi) [inline client/client-multi/client-c/test/platform-impl/test_ext_log_upload_strategies.c]: trailing spaces

**Comment by rasendubi** [comment]
LGTM

@forGGe ?

**Comment by forGGe** [comment]
Looks good

**Pull request merged**